slides/src/content/components.md

```diff
@@ -312,6 +312,63 @@ class ChildComponent {}
 
 ---
 
+## Defining multiple projection areas
+
+- Multiple `<ng-content>` tags can be used in a template by using the `select` attribute
+- In the template, we can use an HTML tag, say, `<header>` to specify the position of projected content to the `ng-content` with `select="header"`
+
+```ts
+  // app.component.ts
+  template: `<rio-child>
+              <header>
+               <p>This is my header content</p>
+              </header>
+              <footer>
+                <p>This is my footer content</p>
+              </footer>
+             </rio-child>`
+```
```

Comment:
I like this idea, maybe we could show html directly which is easier to read because of syntax highlight:

```html
<!-- parent component -->
<rio-child>
  <header><p>This is my header content</p></header>
  <footer><p>This is my footer content</p></footer>
</rio-child>
```

```html
<!-- child component -->
<h4>Child Component</h4>
<ng-content select="header"></ng-content>
<ng-content select="footer"></ng-content>
```